README.md

```diff
@@ -1,273 +1,43 @@
 
-If you'd like to contribute, please see [Contributing.md](https://github.com/apollographql/apollo-kotlin/blob/main/CONTRIBUTING.md).
+Because generated types are operation-specific, you can only access data that you actually specify as part of an operation. If you don't ask for a particular field in an operation, you can't access the corresponding property on the returned data structure.
 
-## Community integrations
+This library is designed with Android in mind, but you can use it in any Kotlin multiplatform application.
 
-* If you're using the [Maven](https://maven.apache.org/) build tool, [apollo-client-maven-plugin](https://github.com/aoudiamoncef/apollo-client-maven-plugin) is a Maven plugin that calls the Apollo Kotlin compiler to generate your Java/Kotlin sources.
-* If you're using [Absinthe Phoenix subscriptions](https://hexdocs.pm/absinthe_phoenix/readme.html), [kotlin-phoenix](https://github.com/ajacquierbret/kotlin-phoenix) has a [PhoenixNetworkTransport](https://github.com/ajacquierbret/kotlin-phoenix/blob/main/kotlinphoenix-adapters/src/commonMain/kotlin/io/github/ajacquierbret/kotlinphoenix/adapters/apollo/PhoenixNetworkTransport.kt) that you can use together with `ApolloClient` ([doc](https://github.com/ajacquierbret/kotlin-phoenix/tree/main/kotlinphoenix-adapters))
+## 📚 Documentation
 
-## Additional resources
+All Apollo Kotlin documentation, including caching and helpful recipes, can be found at: <br/>
+[https://www.apollographql.com/docs/kotlin/](https://www.apollographql.com/docs/kotlin/)
 
-- [Confetti](https://github.com/joreilly/Confetti): A Kotlin Multiplatform conference app using Apollo Kotlin, SwiftUI and Jetpack Compose.
-- [MortyComposeKMM](https://github.com/joreilly/MortyComposeKMM): A Kotlin Multiplatform Github template using Apollo Kotlin, SwiftUI and Jetpack Compose.
-- [A journey to Kotlin multiplatform](https://www.youtube.com/watch?v=GN6LHrqyimI): how the project was moved to Kotlin multiplatform, talk given at Kotliners in June 2020.
-- [#125, Fragmented Podcast](http://fragmentedpodcast.com/episodes/125/): Why's and How's about Apollo Kotlin and the entire journey.
-- [GraphQL.org](http://graphql.org) for an introduction and reference to GraphQL itself.
-- [apollographql.com](http://www.apollographql.com/) to learn about Apollo open-source and commercial tools.
-- [The Apollo blog](https://www.apollographql.com/blog/) for long-form articles about GraphQL, feature announcements for Apollo, and guest articles from the community.
-- [The Apollo Twitter account](https://twitter.com/apollographql) for in-the-moment news.
+The Apollo Kotlin API reference can be found at: <br/>
+[https://apollographql.github.io/apollo-kotlin/kdoc/](https://apollographql.github.io/apollo-kotlin/kdoc/)
 
-## Who is Apollo?
+## 👨‍💻 Who is Apollo?
```

Comment:
The https://www.apollographql.com/studio/develop is 404ing. But I'm wondering if this whole section should be updated (by someone more knowledgeable than me about what it should say 😅), as I think it may not be up-to-date (e.g. no mention of GraphOS).

Reply:
Uh oh, right 👍 , I'll ping the docs team. Also that part:

```
[Apollo Client](https://www.apollographql.com/apollo-client/) – The most popular GraphQL client for the web. Apollo also builds and maintains [Apollo iOS](https://github.com/apollographql/apollo-ios) and [Apollo Kotlin](https://github.com/apollographql/apollo-kotlin).
```

Feels a bit weird. I don't mind the emphasis on apollo-client in general but feels weird to mention Apollo Kotlin as an "also" given the whole page is about Apollo Kotlin.